extensions/ui/org.eclipse.smarthome.ui.paper/about.html

```diff
@@ -80,6 +80,21 @@
 		<a href="https://github.com/choffmeister/roboto-fontface-bower">github/</a> and are made available under the 
 		<a href="about_files/ROBOTO_FONT_LICENSE">Apache License</a>. 
 		</p>
+		
+		<p>
+        <strong>JQueryUI</strong> <br/><br/>
```

Comment:
ok, but you should make sure that the formatting is consistent within this file - so maybe simply edit it with a normal text editor, so that your additions also have tabs instead of spaces.